27.md

```diff
@@ -1,12 +1,51 @@
+
+This project will kick-start a long-term mutually beneficial collaboration between the ELIXIR Tools Platform and the Bioconductor community. 
+
+:dart: **Short-term BioHackathon goals:**
+
+* Mapping of EDAM and biocViews terms
+* “Gold standard" manual annotation of a subset of Bioconductor packages in bio.tools
+* Assessing development or adaptation of a tool for automated EDAM suggestions from biocViews or package content
+
+:dart: **Long-term goals:**
+
+* Extend EDAM to all Bioconductor software packages, and also the thousands of Bioconductor annotation and experiment resources
+* Phase-out biocViews for systematic EDAM annotation
+* Synchronise Bioconductor packages with bio.tools (via automated integration with ELIXIR RSEc)
+
+## Interested in contributing?
+
+:loudspeaker: Reach out! 
+
+* [Bioconductor slack community](https://community-bioc.slack.com/): **#edam-collaboration** 
+* [ELIXIR Europe slack community](https://elixir-europe.slack.com/): **#edam_ontology** 
+
+Our project is committed to inclusivity, guided by the [Bioconductor Code of Conduct](https://www.bioconductor.org/about/code-of-conduct), as well as the [ELIXIR code of conduct for events](https://elixir-europe.org/events/code-of-conduct) and the [ELIXIR RSEc code of conduct](https://github.com/research-software-ecosystem/content/blob/master/CODE_OF_CONDUCT.md). We value inputs from different perspectives - from ontology experts to developers to end user experience - across a diversity of professional, personal, cultural, or linguistic backgrounds.
+
+Remote participation is welcome, and would ideally be planned with the interested parties ahead of the event, to ensure we can get the necessary setup ready to work during the event and have everybody able to fully enjoy and take advantage of the event.
+
+## Project abstract
 
 This project aims to enhance the ELIXIR Research Software Ecosystem (RSEc) by improving the accessibility, interoperability, and reusability of over 2,000 Bioconductor packages. This involves aligning their description with FAIR principles and setting up their synchronisation with the bio.tools registry. Additionally, this project aims to enhance EDAM's utility by applying the EDAM standard to the large Bioconductor ecosystem. The project utilises structured integration processes and community-centric development to achieve these goals. 
 
 It aligns with the ELIXIR 2024-26 program objectives through the standardisation of Bioconductor software metadata, their inclusion in the RSEc infrastructure, and the community-based improvement of EDAM (Tools platform WP2 and WP3). Feasibility is ensured through planned deliverables, including mapping EDAM and biocViews, manual annotation, and exploring automated mapping tools. Long-term plans involve systematic annotation and synchronisation of Bioconductor packages with bio.tools.
 
-## Lead(s)
+## Project leads
```

Comment:
Proposing to keep the heading the same so that it matches the rest of the projects

```suggestion
## Lead(s)
```